MPSGameLift/Scripts/export_gamelift_server_package.py

```diff
@@ -0,0 +1,88 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+"""
+Enables the project for AWS Gamelift and creates a Windows server package which can be uploaded to a GameLift.
+
+To use this script pass it into o3de.bat's export-project command: 
+<path-to-o3de-engine>\scripts\o3de.bat export-project -es <path-to-multiplayer-sample>\MPSGameLift\Scripts\export_gamelift_server_package.py -ll INFO
+"""
+
+import os
+import argparse
+
+import o3de.export_project as exp
+import o3de.enable_gem as enable_gem
+
+from o3de.export_project import process_command
+from o3de import manifest
+
+project_json_data = manifest.get_project_json_data(project_path=o3de_context.project_path)
+project_name = project_json_data.get('project_name')
+
+o3de_logger.info(f"Exporting AWS GameLift Server Package for {project_name}")
+
+# Parse arguments to either build code, assets, or both
+parser = argparse.ArgumentParser(
+                    prog='GameLift Server Package',
+                    description='Helps setup the project for AWS Gamelift and creates a Windows server package which can be uploaded to a GameLift.')
+
+parser.add_argument('--code', action='store_true', help='Build code')
+parser.add_argument('--assets', action='store_true', help='Build assets')
+
+args = parser.parse_args(o3de_context.args)
+
+if not args.code and not args.assets:
+    user_input = input('No build command specified. Do you want to build code, assets, or both? (c/a/b): ')
```

Comment:
I'm thinking it'd be good to have this in a validation while loop, otherwise the variables are not set if the user doesn't press the correct key.

Also should we allow a `q` key, to quit if the user changes their mind?

Reply:
Thanks! Updated!